core/domain/email_manager.py

```diff
@@ -561,13 +561,13 @@ def send_emails_to_subscribers(creator_id, exploration_id, exploration_title):
     """
 
     creator_name = user_services.get_username(creator_id)
-    email_subject = ('%s has published another exploration!' % creator_name)
+    email_subject = ('%s has published a new exploration!' % creator_name)
     email_body_template = (
         'Hi %s,<br>'
         '<br>'
-        'Your subscription, %s has published a new exploration. You can play '
-        'the exploration over here - '
-        '<a href="https://www.oppia.org/explore/%s">%s</a>.<br>'
+        '%s has published a new exploration, %s. Play it here: <br>'
```

Comment:
Ah, sorry! I misread the PR before, and didn't realize the exploration title was in the link; that would actually read better:

    '%s has published a new exploration! You can play it here: '
    '<a href="https://www.oppia.org/explore/%s">Exploration Title</a>.<br>'

Sorry for the error!